test/WebJobs.Script.Tests/Configuration/FunctionsHostingConfigurationTest.cs

```diff
@@ -134,5 +127,12 @@ public void ReturnsExpectedConfig(string config, int configCount)
                 Assert.True(conf.Config.Count == configCount);
             }
         }
+
+        [Fact]
+        public void GetValue_ConfigurationFileDoesNotExists()
```

Comment:
nit: in this test, the configuration file DOES actually exist right? Should it be named something like Constructor_ConfigFileExists_DoesNotLog? What about the other test - Constructor_ConfigFileDoesNotExist_Logs